deploy/src/native_to_erc/foreign.js

```diff
@@ -195,6 +199,36 @@ async function deployForeign() {
   assert.equal(Web3Utils.hexToNumber(setBridgeContract.status), 1, 'Transaction Failed')
   foreignNonce++
 
+  if (DEPLOY_REWARDABLE_TOKEN) {
+    console.log('\nset BlockReward contract on ERC677BridgeTokenRewardable')
```

Comment:
A quick question...
Will we consider the rewardable token deployed correctly if it has neither the block reward contract nor the validator set contract defined? If it is critical does it make sense to modify the token contract as so to block all operation until the contracts are defined (or initialize all needed contracts addresses in the token constructor)?

Reply:
No, it doesn't make sense because `ERC677BridgeTokenRewardable` contract works just like `ERC677BridgeToken` if `setBlockRewardContract` and/or `setValidatorSetContract` functions are nevel called.

Nothing will be broken if we deploy `ERC677BridgeTokenRewardable`, use it for some time and then call `setBlockRewardContract` and/or `setValidatorSetContract` and continue using.

Reply:
thanks for clarification